labman/db/process.py

```diff
@@ -2968,7 +2968,7 @@ def _generate_amplicon_sample_sheet(self):
                 % ('Lane,' if self.include_lane else '')]
         for pool, lane in self.pools:
             data.append('%s%s,,,,,NNNNNNNNNNNN,,%s,,,'
-                        % (('%s,' % (lane if self.include_lane else '')),
+                        % (('%s,' % lane) if self.include_lane else '',
                            self._bcl_scrub_name(pool.container.external_id),
```

Comment:
I do not understand this change.  @wasade requested "enclose lane if self.include_lane else '' in parentheses", which is not the change that was made here.  In fact, now the ternary conditional (lane if self.include_lane else '') is broken up by a parenthesis, which just doesn't seem right.  @wasade , can you weigh in?

Reply:
Hi @AmandaBirmingham , the edit Daniel suggested will produce a different result. But his rationale was right, that my original version was visually confusing. Therefore I edited this part to make it visually clean.

Reply:
The correct logic is:
```
if self.include_lane:
    return '%s,' % lane
else:
    return ''
```

Reply:
Ahh!  Now I gotcha!  Thank you for clarifying for me :)